index.html

```diff
@@ -1,138 +1,150 @@
-      </a>
-    </li>
-      <li class="navigation-item">
-        <a class="navigation-link" href="#">
-          <span class="visually">
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Седона</title>
+</head>
+
+<body>
+
+  <header class="page-header">
+    <nav class="navigation">
+      <ul class="navigation-list">
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">Главная</a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">О Седоне</a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="catalog.html">Гостиницы</a>
+        </li>
+      </ul>
+      <img class="logo" alt="логотип city of Sedona">
+      <ul class="navigation-list navigation-user">
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">
+            <span class="visually-hidden">
+              Поиск
+            </span>
+          </a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">
+            <span class="visually-hidden">
+              Избранное
+            </span>
+          </a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">
             Хочу сюда
-        </span>
-      </a>
-  </li>
-  </ul>
-</nav>
-</header>
-<main class="main-index">
-<h1 class="visually-hidden">Седона</h1>
-<img class="welcome" src="" alt="Добро пожаловать в Седону">
-<section class="advantages">
-<ul class="advantages-list">
-  <li class="advantages-item">
-    <h2>Седона — небольшой городок в Аризоне, заслуживающий большего!</h2>
-    <span>Рассмотрим 5 причин, по которым Седона круче, чем гранд каньон!</span>
-    <h3 class="advantages-title"> Настоящий городок</h3>
-    <p class="advantages-description">— №1 — Седона — не аттракцион для туристов, там течёт
-      своя жизнь</p>
-      <img src="" alt="городок">
-      <ul class="features-list">
-        <li class="features-iteam">
-        <img class="icon" src="" alt="Жильё">
-        <h3 class="features-title">Жильё</h3>
-        <p class="features-description">Рекомендуем пожить<br> в настоящем мотеле,<br> всё как в кино!</p>
+          </a>
         </li>
-        <li class="features-iteam">
-          <img class="icon" src="" alt="Еда">
-          <h3 class="features-title">Еда</h3>
-          <p class="features-description">Всегда заказывайте<br> топовый фирменный бургер,<br> вы не разочаруетесь!</p>
-          </li>
-          <li class="features-iteam">
-            <img class="icon" src="" alt="Сувениры">
-            <h3 class="features-title">Сувениры</h3>
-            <p class="features-description">Не только китайского,<br> но и настоящего местного<br> производства!</p>
+      </ul>
+    </nav>
+  </header>
+  <main class="main-index">
+    <h1 class="visually-hidden">Седона</h1>
+    <img class="welcome" src="" alt="welcome to the gargeous Sedona">
```

Comment:
```suggestion
    <img class="welcome" src="" alt="welcome to the gorgeous Sedona">
```

Хотя текст там длиннее (и немного нецензурный, скажем прямо)

Reply:
Там дальше кавычки, я не хотела путать в разметке и писать большой текст к картинке. Или лучше добавить? Добавила)